python/grass/script/task.py

```diff
@@ -502,7 +502,7 @@ def get_interface_description(cmd):
                 "Unable to fetch interface description for command '<{cmd}>'."
                 "\n\nDetails: <{det}>"
             ).format(cmd=cmd, det=e)
-        )
+        ) from e
```

Comment:
Here is another (of several) examples, where the original Error messages is included in the error message that is raised at the end. So, I guess we also here could replace the formating with exception chaining?

Reply:
Maybe this one with a `from None`?

Reply:
The motivation for these `Details:` is to give user the original exception. This would be better done with chained exceptions in Python. However, the code was written with the expectation that the error will be read by a command line tool user without a traceback (or the original exception). Using `from None` preserves that idea, but I would be open to a solution which goes the path with less expectations about the potential usage.

Perhaps `from e` combined with the `Details:` has most information when doing `print(...)` and when just looking at the traceback.